common/components/Header/components/NavigationLink.tsx

```diff
@@ -10,12 +10,19 @@ interface Props {
     to?: string;
     external?: boolean;
   };
-  location: any;
 }
 
-export default class NavigationLink extends React.Component<Props, {}> {
+interface InjectedLocation extends Props {
+  location: { pathname: string };
+}
+
+class NavigationLink extends React.Component<Props, {}> {
```

Comment:
Couldn't we use `InjectedLocation` instead of Props here, and avoid creating the `injected` method?

Reply:
yes we can, ill fix that